engine/access/access_test.go

```diff
@@ -92,6 +94,7 @@ func TestAccess(t *testing.T) {
 }
 
 func (suite *Suite) SetupTest() {
+	suite.lockManager = storage.NewTestingLockManager()
```

Comment:
SetupTest is called for each test case, we create the single lock manager at the beginning to ensure it's only created once.